_data/pages.yaml

```diff
@@ -545,8 +545,17 @@
                   id: "reasons-for-withdrawing"
                 - title: "Steps to retiring an extension"
                   id: "steps-to-retiring-an-extension"
-                - title: Suggested retirement timetable
+                - title: "Suggested retirement timetable"
                   id: "suggested-retirement-timetable"
+            - title: "Updating your extension"
+              url: "/documentation/manage/updating-your-extension/"
+              subpageitems:
+                - title: "Enabling updates to your extension"
+                  id: "enabling-updates-to-your-extension"
```

Comment:
Very close! Swap out `enabling-updates-to-your-extension` for `enable-updates` and this should be good to go.